doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst

```diff
@@ -0,0 +1,168 @@
+
+All functionality related to nRF54H20 SoC is considered experimental.
+
+Migration notes
+***************
+
+See the `Migration guide for nRF Connect SDK v2.6.99_cs2 for v2.4.99-cs3 users`_ for the changes required or recommended when migrating your application from v2.4.99-cs3 to v2.6.99-cs2.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Peripherals support
+===================
+
+* Added experimental support for the following peripherals on the nRF54H20 DK:
+
+  * ADC
+  * CLOCK
+  * GPIO
+  * GRTC
+  * I2C
+  * MRAM
+  * NFCT
+  * PWM
+  * RADIO (Bluetooth Low Energy and 802.15.4)
+  * SPI
+  * UART
+  * WDT
+
+Working with nRF54H Series
+==========================
+
+* Added:
+
+  * The :ref:`ug_nrf54h20_gs` page.
+
+..
+  ##ADD the rest of the doc pages
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Enhanced ShockBurst (ESB)
+-------------------------
+
+* Added:
+
+  * Support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` board.
+  * Fast switching between radio states for the nRF54H20 SoC.
+  * Fast radio channel switching for the nRF54H20 SoC.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Bluetooth samples
+-----------------
+
+* :ref:`direct_test_mode` sample:
+
+  * Added:
+
+    * Support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build target ``nrf54h20dk_nrf54h20_cpurad``.
+    * Experimental support for the HCI interface.
+
+  * Updated the internal sample API.
+
+Enhanced ShockBurst samples
+---------------------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build target ``nrf54h20dk_nrf54h20_cpurad`` in the following ESB samples:
+
+  * :ref:`esb_prx` sample
+  * :ref:`esb_ptx` sample
+
+Multicore samples
+-----------------
+
+* :ref:`multicore_hello_world` sample:
+
+  * Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build target ``nrf54h20dk_nrf54h20_cpuapp``.
+
+Peripheral samples
+------------------
+
+* :ref:`radio_test` sample:
+
+  * Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build target ``nrf54h20dk_nrf54h20_cpurad``.
+  * Updated the CLI command ``fem tx_power_control <tx_power_control>`` which replaces ``fem tx_gain <tx_gain>`` .
+    This change applies to the sample built with the Kconfig option :ref:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC <CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC>` set to ``n``.
+
+Zephyr
+======
+
+..
+  ##update commits
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``0051731a41fa2c9057f360dc9b819e47b2484be5``, with some |NCS| specific additions.
```

Comment:
![image](https://github.com/nrfconnect/sdk-nrf/assets/125915901/b357efbd-b775-49d2-bfda-fb3562c44422)

Reply:
@rlubos

Reply:
This looks correct, it corresponds to the merge base used in the last upmerge on this branch.